src/main/java/com/minecolonies/coremod/colony/buildings/modules/LivingBuildingModule.java

```diff
@@ -247,7 +247,7 @@ else if (!data.isChild())
             final int populationCount = building.getColony().getCitizenManager().getCurrentCitizenCount();
             AdvancementUtils.TriggerAdvancementPlayersForColony(building.getColony(), playerMP -> AdvancementTriggers.COLONY_POPULATION.trigger(playerMP, populationCount));
 
-            LanguageHandler.sendPlayersMessage(building.getColony().getImportantMessageEntityPlayers(), "com.minecolonies.coremod.progress.newChild");
+            LanguageHandler.sendPlayersMessage(building.getColony().getImportantMessageEntityPlayers(), "com.minecolonies.coremod.progress.newChild", building.getColony());
```

Comment:
not getName?

Reply:
You said in discord to do `building.getColony()` 😛

Reply:
for the colony, but then additionally you gotta do getName() xD for the name